apps/judicial-system/web/src/components/Header/Header.tsx

```diff
@@ -113,11 +113,47 @@ const HeaderContainer = () => {
     api.logout()
   }
 
-  const handleChangeInstitution = () => {
-    router.push('/')
+  const handleSelectUser = async (userId?: string) => {
+    if (userId) {
+      await api.activate(userId)
+    } else {
+      router.push('/')
+    }
+
     setIsUserMenuOpen(false)
   }
 
+  const renderSelectUser = () => {
+    if (!user || !eligibleUsers || eligibleUsers.length < 2) {
+      return null
+    }
+
+    const otherUser =
+      eligibleUsers.length === 2
+        ? eligibleUsers.find((u) => u.id !== user.id)
+        : null
+
+    return (
+      <Box marginTop={2}>
+        <Button
+          variant="text"
+          onClick={() => handleSelectUser(otherUser?.id)}
+          size="small"
+          preTextIcon="swapHorizontal"
+        >
+          {otherUser
+            ? otherUser.institution?.name
+            : 'Skipta um embætti / hlutverk'}
+        </Button>
+        {otherUser && (
```

Comment:
Did you run this by Guðný? I feel like this is a bit "out of place" in the user menu. What do you think?

Reply:
Yes I did. We are not always changing institution, we may have another user in the same institution with a different role.

Reply:
Showing the other user's title clarifies to the user what the other user does.